atuin/src/command/client/search/interactive.rs

```diff
@@ -580,12 +580,26 @@ impl State {
             }
 
             1 => {
-                super::inspector::draw(
-                    f,
-                    results_list_chunk,
-                    &results[self.results_state.selected()],
-                    &stats.expect("Drawing inspector, but no stats"),
-                );
+                if results.is_empty() {
+                    let message = Paragraph::new("Nothing to inspect. :-(")
```

Comment:
```suggestion
                    let message = Paragraph::new("Nothing to inspect")
```

Reply:
Oops, sorry. orhun uses `:(` in some of the messages in rustypaste. I kinda liked it and it must have stuck...